devbox.go

```diff
@@ -379,6 +381,73 @@ func (d *Devbox) Info(pkg string) error {
 	return pkgcfg.PrintReadme(pkg, d.configDir, d.writer, false /*showSourceEnv*/)
 }
 
+// generates devcontainer.json and Dockerfile for vscode run-in-container
+// and Github Codespaces
+func (d *Devbox) GenerateDevcontainer(force bool) error {
+	// construct path to devcontainer directory
+	devContainerPath := filepath.Join(d.configDir, ".devcontainer/")
+	devContainerJSONPath := filepath.Join(devContainerPath, "devcontainer.json")
+	dockerfilePath := filepath.Join(devContainerPath, "Dockerfile")
+
+	filesExist := false
+	// check if devcontainer.json doesn't exits
+	if plansdk.FileExists(devContainerJSONPath) {
+		filesExist = true
+	}
+	// check if Dockerfile doesn't exits
+	if plansdk.FileExists(dockerfilePath) {
+		filesExist = true
+	}
```

Comment:
Should we just say, if the directory `.devcontainer` exist, then files exist?

Reply:
I initially did that, but I realized if users have other files in there, it will be annoying to have to delete the folder just to generate the devcontainer files.